subsys/net/lib/sockets/Kconfig

```diff
@@ -25,6 +25,13 @@ config NET_SOCKETS_POSIX_NAMES
 	attention, as in POSIX it closes any file descriptor, while with this
 	option enaled, it will still apply only to sockets.
 
+config NET_SOCKETS_POLL_NUM
```

Comment:
Let me grep thru Kconfigs and see how it's usually called, somehow I thought the convention is *_NUM.